### ISSUE TITLE
Embedded images not showing

### ISSUE THREAD

**Issue by Fenisu** [nilsbraden/ttrss-reader-fork#309]
The images embedded on some feeds does not show on this android app. It appears an empty small square, or a smiliar to a typical "IE" old pic-not-loaded icon. I am willing to help debugging the problem but I don't know how. The rss feed is this (a crappy, site I know...): 
https://feeds.feedburner.com/fayerwayer

**Comment by nilsbraden** [comment]
Can you please try to disable the preference SSL Settings -> Use Play Services...? It seems like google has some concerns about the certificates by COMODO CA Limited which are used by static.betazeta.com, the image hosting server of this feed.

**Comment by Fenisu** [comment]
Yes, that was it, though the problem partialy persists. For some reason, the images offered in the middle of the posts (those not in the header) are not https links (just plain http), and it appears that the app refuses to load them.
